packages/@lwc/node-reactions/src/dom-patching/node.ts

```diff
@@ -30,9 +30,9 @@ export default function() {
                     return appendChild.call(this, aChild);
                 }
 
-                const qualifiedReactionTypes: Array<ReactionType> = [];
+                const qualifiedReactionTypes: QualfyingReactionTypes = {};
```

Comment:
make this obj explicit shape... even if it is set to false for connect and disconnect. another alternative is to provide a javascript bitwise flag that can be compared easily.